blockchain/types/tx_internal_data_ethereum_set_code.go

```diff
@@ -561,6 +561,14 @@ type SetCodeAuthorization struct {
 	S       uint256.Int    `json:"s"`
 }
 
```

Comment:
```suggestion
//go:generate go run github.com/fjl/gencodec -type SetCodeAuthorization -field-override authorizationMarshaling -out gen_authorization.go
type SetCodeAuthorization struct {
	ChainID uint256.Int    `json:"chainId"`
	Address common.Address `json:"address"`
	Nonce   uint64         `json:"nonce"`
	V       uint8          `json:"yParity"`
	R       uint256.Int    `json:"r"`
	S       uint256.Int    `json:"s"`
}
```
Would you fix the original struct as well?